apps/community/habitat.yaml

```diff
@@ -0,0 +1,11 @@
+name: community
+bin: community
+ports:
+  - 8008
+proxy_rules:
+  - type: file
+    matcher: /community
+    target: web
+  - type: redirect
```

Comment:
is this new? are the proxy rules documented anywhere?

Reply:
Nope, we've had this for a long time. Not documented.